dali/operators/math/expressions/expression_tile.h

```diff
@@ -31,13 +31,12 @@ namespace dali {
  */
 struct TileDesc {
   int sample_idx;       // id of sample inside within the batch
-  int extent_idx;       // the index of tile within this sample_idx
-  int64_t extent_size;  // actually covered extent in this tile, the last can be smaller
-  int64_t tile_size;    // the size of regular tile
+  int64_t offset;
+  int64_t extent_size;
```

Comment:
Nitpick: just one of "extent" or "size"?
```suggestion
  int64_t extent;
```